pkg/apis/kudo/v1beta1/instance_types_helpers.go

```diff
@@ -196,6 +202,28 @@ func (i *Instance) IsTopLevelInstance() bool {
 	return !i.IsChildInstance()
 }
 
+const (
+	planInProgressReason   = "PlanInProgress"
+	resourceNotReadyReason = "ResourceNotReady"
+	readyConditionType     = "Ready"
+)
+
+func (i *Instance) SetReadinessTrue(msg string) {
+	condition := metav1.Condition{Type: readyConditionType, Status: metav1.ConditionTrue, Message: msg}
```

Comment:
Looks like we always have to set `Reason` to a non-empty string. The CI is failing with logs like
> InstanceController: Error when updating instance status. Instance.kudo.dev "toy1" is invalid: status.conditions.reason: Invalid value: "": status.conditions.reason in body should be at least 1 chars long